src/components/Avatar/Avatar.vue

```diff
@@ -171,6 +171,13 @@ export default {
 			type: Boolean,
 			default: true,
 		},
+		/**
+		 * When the user status was preloaded via another source it can be handed in with this property to save the request.
+		 */
+		preloadedUserStatus: {
```

Comment:
Not sure if we even need to call it "preloaded". Technically, all properties are preloaded.

We could say in the doc that if this property is not set, its data will be fetched automatically from the server.

Reply:
userStatus is already a data attribute in the mixin and I didn't want to mess around with that.

Reply:
Added more docs

Reply:
ah, indeed.